src/SparkTestBase.sol

```diff
@@ -334,6 +343,34 @@ abstract contract SparkEthereumTestBase is SparkTestBase {
         _assertPaused(WETH, true);
     }
 
+    function _assertBorrowCapConfig(address asset, uint256 max, uint256 gap, uint256 increaseCooldown) internal {
+        (uint256 _max, uint256 _gap, uint256 _increaseCooldown,,) = capAutomator.borrowCapConfigs(asset);
```

Comment:
Solidity doesn't pack these, but the sizing is actually `uint48`. They should be updated to this just to be extra cautious.